a10_octavia/common/a10constants.py

```diff
@@ -19,23 +19,50 @@
 ROLE = 'role'
 BACKUP_VTHUNDER = 'backup_vthunder'
 VRRP_STATUS = 'vrrp_status'
+MASTER_VRRP_STATUS = 'master_vrrp_status'
+BACKUP_VRRP_STATUS = 'backup_vrrp_status'
 
 FAILED = 'FAILED'
 USED_SPARE = 'USED_SPARE'
 OCTAVIA_HEALTH_MANAGER_CONTROLLER = 'octavia_health_manager_controller'
 OCTAVIA_HEALTH_MONITOR = 'octavia_health_monitor'
 VTHUNDER_CONFIG = 'vthunder_config'
 VTHUNDER_ID = "vthunder_id"
+DEVICE_ID = "device_id"
+SET_ID = "set_id"
+VRID = "vrid"
+DEVICE_PRIORITY = 'device_priority'
+FLOATING_IP = 'floating_ip'
+FLOATING_IP_MASK = 'floating_ip_mask'
 SP_OBJ_DICT = {
     'HTTP_COOKIE': "cookie_persistence",
     'APP_COOKIE': "cookie_persistence",
     'SOURCE_IP': "src_ip_persistence",
 }
+HM_TYPE = ['HTTP', 'HTTPS']
 PERS_TYPE = ['cookie_persistence', 'src_ip_persistence']
 # Taskflow flow and task names
 
+BACKUP_AMPHORA_PLUG = 'backup-amphora-plug'
+BACKUP_CONNECTIVITY_WAIT = 'backup-connectivity-wait'
+BACKUP_ENABLE_INTERFACE = 'backup-enable-interface'
 MARK_VTHUNDER_MASTER_ACTIVE_IN_DB = 'mark-vthunder-master-active-in-db'
 MARK_VTHUNDER_BACKUP_ACTIVE_IN_DB = 'mark-vthunder-backup-active-in-db'
 GET_BACKUP_VTHUNDER_BY_LB = 'get-backup-vthunder-by-lb'
 CREATE_HEALTH_MONITOR_ON_VTHUNDER_MASTER = 'create-health-monitor-on-vthunder-master'
 HANDLE_SESS_PERS = 'handle-session-persistence-delta-subflow'
+GET_LOADBALANCER_FROM_DB = 'Get-Loadbalancer-from_db'
+GET_BACKUP_LOADBALANCER_FROM_DB = 'get-backup-loadbalancer-from-db'
+CONFIGURE_VRRP_FOR_MASTER_VTHUNDER = 'configure-vrrp-for-master-vthunder'
+CONFIGURE_VRRP_FOR_BACKUP_VTHUNDER = 'configure-vrrp-for-backup-vthunder'
+CONFIGURE_VRID_FOR_MASTER_VTHUNDER = 'configure-vrid-for-master-vthunder'
+CONFIGURE_VRID_FOR_BACKUP_VTHUNDER = 'configure-vrid-for-backup-vthunder'
+CONFIGURE_VRRP_SYNC = 'configure-vrrp-sync'
+WAIT_FOR_MASTER_SYNC = 'wait-for-master_sync'
```

Comment:
Both of these should be `wait-for-master-sync` and `wait-for-backup-sync`

Reply:
Correction completed